__tests__/utils.js

```diff
@@ -34,9 +34,14 @@ describe('getFlagArrayFromObject', () => {
 
   test('correctly deconstructs ignoredFiles flag', () => {
     const input = {
-      ignoredFiles: ['file1', 'file2']
+      ignoredFile: ['file1', 'file2']
```

Comment:
`ignoredFiles` has changed to `ignoredFile`? If so thats a breaking change. Can we keep it `ignoredFiles`?

Reply:
The themekit flag is --ignored-file and our node-themekit documentation says

> All flags specified in the Theme Kit documentation are available, but in camelCase rather than in --flag-form

We can make it ignoredFiles again but then we'd have to start making a list of exceptions as to which flags arent exactly the camelCase version of the themekit docs which doesn't seem ideal

Reply:
Okay, I like your logic -- we'll just need to do a major version bump since this is breaking.